test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -190,126 +190,121 @@ def test_make_xsd_id_compatible(self) -> None:
-        self.assertEqual(excel2xml.find_date_in_string("x (5/11/2021) x"), "GREGORIAN:CE:2021-11-05:CE:2021-11-05")
-        self.assertEqual(excel2xml.find_date_in_string("x ...2193_01_26... x"), "GREGORIAN:CE:2193-01-26:CE:2193-01-26")
-        self.assertEqual(excel2xml.find_date_in_string("x -2193_01_26- x"), "GREGORIAN:CE:2193-01-26:CE:2193-01-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 2193_02_30 x"), None)
+        assert excel2xml.find_date_in_string("x (30.4.2021) x") == "GREGORIAN:CE:2021-04-30:CE:2021-04-30"
+        assert excel2xml.find_date_in_string("x (5/11/2021) x") == "GREGORIAN:CE:2021-11-05:CE:2021-11-05"
+        assert excel2xml.find_date_in_string("x ...2193_01_26... x") == "GREGORIAN:CE:2193-01-26:CE:2193-01-26"
+        assert excel2xml.find_date_in_string("x -2193_01_26- x") == "GREGORIAN:CE:2193-01-26:CE:2193-01-26"
+        assert excel2xml.find_date_in_string("x 2193_02_30 x") is None
 
     def test_find_date_in_string_eur_date_range(self) -> None:
         """template: 27.-28.1.1900"""
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-26.2.0800 x"), "GREGORIAN:CE:0800-02-25:CE:0800-02-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 25. - 26.2.0800 x"), "GREGORIAN:CE:0800-02-25:CE:0800-02-26")
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-24.2.0800 x"), None)
-        self.assertEqual(excel2xml.find_date_in_string("x 25.-25.2.0800 x"), None)
+        assert excel2xml.find_date_in_string("x 25.-26.2.0800 x") == "GREGORIAN:CE:0800-02-25:CE:0800-02-26"
+        assert excel2xml.find_date_in_string("x 25. - 26.2.0800 x") == "GREGORIAN:CE:0800-02-25:CE:0800-02-26"
+        assert excel2xml.find_date_in_string("x 25.-24.2.0800 x") is None
+        assert excel2xml.find_date_in_string("x 25.-25.2.0800 x") is None
 
     def test_find_date_in_string_eur_date_range_across_month(self) -> None:
         """template: 26.2.-24.3.1948"""
-        expected = "GREGORIAN:CE:2022-03-01:CE:2022-04-25"
-        self.assertEqual(excel2xml.find_date_in_string("x _1.3. - 25.4.2022_ x"), expected)
-        self.assertEqual(excel2xml.find_date_in_string("x (01.03. - 25.04.2022) x"), expected)
+        assert excel2xml.find_date_in_string("x _1.3. - 25.4.2022_ x") == "GREGORIAN:CE:2022-03-01:CE:2022-04-25"
+        assert excel2xml.find_date_in_string("x (01.03. - 25.04.2022) x") == "GREGORIAN:CE:2022-03-01:CE:2022-04-25"
 
-        self.assertEqual(excel2xml.find_date_in_string("x 28.2.-1.12.1515 x"), "GREGORIAN:CE:1515-02-28:CE:1515-12-01")
-        self.assertEqual(excel2xml.find_date_in_string("x 28.2.-26.2.1515 x"), None)
-        self.assertEqual(excel2xml.find_date_in_string("x 28.2.-28.2.1515 x"), None)
+        assert excel2xml.find_date_in_string("x 28.2.-1.12.1515 x") == "GREGORIAN:CE:1515-02-28:CE:1515-12-01"
+        assert excel2xml.find_date_in_string("x 28.2.-26.2.1515 x") is None
+        assert excel2xml.find_date_in_string("x 28.2.-28.2.1515 x") is None
```

Comment:
Same as above

Reply:
I will do that as follow-up: https://linear.app/dasch/issue/DEV-3154/featexcel2xml-find-date-in-string-allow-time-spans-of-1-day